### PR TITLE
`vim bro` a new and simple extension that helps you learn vim commands by letting you search them in raycast.

### PR DESCRIPTION
**Pull request by ajaypremshankar**
## Description

**Vim Bro** is a simple extension that can help learn vim quickly. 
It shows a list of vim commands and you can search commands without leaving the current window.

## Screencast

Screenshot 2022-03-18 at 11 19 39 PM
Screenshot 2022-03-18 at 11 20 04 PM
Screenshot 2022-03-18 at 11 20 35 PM

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed in the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @ajaypremshankar 👋 

it seems like you have the `Trello` extension within your PR, could you please remove this, so it only contains `vim-bro`

We added support for [categories](https://developers.raycast.com/basics/prepare-an-extension-for-store#categories), [screenshots](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) and [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) – do you have time for adding this too

The images in the README should be placed in a folder called `media´ too

I'll do a bigger review when you have addressed above 💪 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by ajaypremshankar** [comment]
> Hi @ajaypremshankar 👋
> 
> it seems like you have the `Trello` extension within your PR, could you please remove this, so it only contains `vim-bro`
> 
> We added support for [categories](https://developers.raycast.com/basics/prepare-an-extension-for-store#categories), [screenshots](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) and [changelog](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) – do you have time for adding this too
> 
> The images in the README should be placed in a folder called `media´ too
> 
> I'll do a bigger review when you have addressed above 💪
> 
> Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

Made changes @pernielsentikaer 

> The images in the README should be placed in a folder called `media´ too
Can I reuse files from `metadata` folder itself, instead of duplicating them in `media` folder?

File: extensions/vim-bro/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      enableFiltering={false}
+      onSearchTextChange={setSearchText}
+      navigationTitle="Search vim commands"
+      searchBarPlaceholder="Learn new command by searching it here."
+    >
+      {filteredList.map((commandGroup: CommandGroup) => {
+        const key = commandGroup.key;
+        const commands = commandGroup.commands;
+        return (
+          <List.Section title={key[0].toUpperCase() + key.slice(1)}>
```
Comment (pernielsentikaer) [inline extensions/vim-bro/src/index.tsx:60]:
```suggestion
          <List.Section key={key} title={key[0].toUpperCase() + key.slice(1)}>
```

File: extensions/vim-bro/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            {commands.map((command: Command) => {
+              return <List.Item title={command.text[0].toUpperCase() + command.text.slice(1)} subtitle={command.kbd} />;
```
Comment (pernielsentikaer) [inline extensions/vim-bro/src/index.tsx:62]:
```suggestion
              return <List.Item key={command.kbd} title={command.text[0].toUpperCase() + command.text.slice(1)} subtitle={command.kbd} />;
```

**Review by pernielsentikaer — COMMENTED** [review]
Hi

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @ajaypremshankar 👋 

Thanks for the update 🔥 

- The metadata images are not in the right format as stated in the guidelines for [screenshots](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots)

- The current images should be moved to `media` folder and referenced in the README 

- Each child in a list should have a unique “key” prop – as suggested, you're missing key attributes 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by ajaypremshankar** [comment]
> Hi @ajaypremshankar 👋
> 
> Thanks for the update 🔥 It's much better 💪 💪 💪
> 
> I have some feedback for you to address 💯
> 
> * The metadata images are not in the right format as stated in the guidelines for [screenshots](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots)
> * The current images should be moved to `media` folder and referenced in the README
> * Each child in a list should have a unique “key” prop – as suggested, you're missing key attributes 😄
> 
> Looking forward to testing it out again 😄
> 
> Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

@pernielsentikaer 
Done, done and done 😄

File: extensions/vim-bro/src/index.tsx (hunk 2 above)
Comment (ajaypremshankar) [reply extensions/vim-bro/src/index.tsx]: Done

**Pull request merged**